python/ray/scripts/scripts.py

```diff
@@ -759,90 +772,71 @@ def start(
-                    bootstrap_address,
-                )
-                cli_logger.newline()
-            if ray_constants.ENABLE_RAY_CLUSTER:
+
+            cli_logger.newline()
+            cli_logger.print(
+                "To connect to this Ray cluster, run `ray.init()` as usual:"
+            )
+            with cli_logger.indented():
+                cli_logger.print("{} ray", cf.magenta("import"))
                 cli_logger.print(
-                    "To connect to this Ray cluster, run `ray.init()` as usual:"
-                )
-                with cli_logger.indented():
-                    cli_logger.print("{} ray", cf.magenta("import"))
-                    cli_logger.print(
-                        "ray{}init()",
-                        cf.magenta("."),
+                    "ray{}init({})",
+                    cf.magenta("."),
+                    "_node_ip_address{}{}".format(
+                        cf.magenta("="), cf.yellow("'" + node_ip_address + "'")
                     )
-                cli_logger.newline()
-                cli_logger.print(
-                    "To connect to this Ray instance from outside of "
-                    "the cluster, for example "
+                    if include_node_ip_address
+                    else "",
                 )
+            cli_logger.newline()
+            cli_logger.print(
+                "To connect to this Ray instance from outside of "
+                "the cluster, for example "
+            )
+            cli_logger.print(
+                "when connecting to a remote cluster from your laptop, " "make sure the"
+            )
+            cli_logger.print(
+                "dashboard {}is accessible and use the Ray Jobs API. For example:",
+                f"({dashboard_url}) " if dashboard_url else "",
+            )
+            if dashboard_url:
                 cli_logger.print(
-                    "when connecting to a remote cluster from your laptop, "
-                    "make sure the"
+                    cf.bold(
+                        "  RAY_ADDRESS='http://<dashboard URL>:{}' ray job submit "
+                        "--working-dir . "
+                        "-- python my_script.py"
+                    ),
+                    ray_params.dashboard_port,
                 )
+            cli_logger.newline()
+            cli_logger.print(
+                "See https://docs.ray.io/en/latest/cluster/running-applications"
+                "/job-submission/index.html"
+            )
+            cli_logger.print(
+                "for more information on connecting to the Ray cluster from "
+                "a remote client."
+            )
+            cli_logger.newline()
+            cli_logger.print("To see the status of the cluster, use")
+            cli_logger.print("  {}".format(cf.bold("ray status")))
+            if dashboard_url:
+                cli_logger.print("To monitor and debug Ray, view the dashboard at ")
```

Comment:
Maybe add a blank line between ray status section and the dashboard section as the other paragraphs?

Reply:
On Anyscale, will the dashboard address be printed correctly?

Reply:
On Anyscale, are we able to see this output? I have never seen it.

Reply:
Let's ignore the anyscale case for now.